dev-requirements-py3.txt

```diff
@@ -1,14 +1,14 @@
 flake8==3.8.3
 bandit==1.6.2
-demisto-py==2.0.11
+demisto-py>=2.0.11
 demisto-sdk==1.2.6
 pytest==6.0.2
 pytest-mock==3.3.1
 requests-mock==1.8.0
 ruamel.yaml==0.16.5
 pipenv>=2020.8.13
 autopep8==1.5.4
-mypy==0.782
+mypy==0.790
```

Comment:
This change I think fails our `lint` as they added additional testing logic. I would do this in a separate PR to deal with all the fixes.

Reply:
Done